onnx2keras.py

```diff
@@ -42,6 +42,18 @@ def ensure_data_format(tensor, format):
             warnings.warn("Transpose inserted. Please report at https://github.com/AxisCommunications/onnx-to-keras/issues", OptimizationMissingWarning)
         out.data_format = InterleavedImageBatch
         return out
+    elif tensor.data_format is InterleavedImageBatch and format is OnnxConstant:
+        assert len(tensor.shape) == 4
+        n, h, w, c = tensor.shape
+        if h == w == 1 or c == 1:
```

Comment:
Why is this extra case necessary? Do you have an example where this can occur?

Reply:
I looked at my use cases again and didn't find one where this occurs, so it is probably not needed.